### PR TITLE
Rename Service port to match the name of the port in the agent container

### PR DESCRIPTION
**Pull request by gaffneyd4**
#### What this PR does / why we need it:

In order for [CiliumLocalRedirectPolicy's](https://docs.cilium.io/en/v1.12/gettingstarted/local-redirect-policy) to work, port names in Services and Pod specs must be named the same.
I did not find any references to the Service port name in the helm chart, so renaming the port in the Service definition should not be a breaking change.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] Chart Version bumped
- [x] `CHANGELOG.md` has been updated
- [x] Variables are documented in the `README.md`

### CODE REVIEW

**Comment by gaffneyd4** [comment]
This failure `Lint and Test Charts / install-chart (v1.22.12) (pull_request)` looks unrelated to my change...

> Command Output: Error response from daemon: manifest for kindest/node:v1.22.12 not found: manifest unknown: manifest unknown

**Review by kayayarai — APPROVED** [review]

**Comment by gaffneyd4** [comment]
Force pushed to fix merge conflicts. I see this PR has an approval. Is this okay to merge?

**Comment by clamoriniere** [comment]
Hi @gaffneyd4 

Thanks for opening this pull request. 

>  I did not find any references to the Service port name in the helm chart, so renaming the port in the Service definition should not be a breaking change.

The problem can be in APM/Dogstastd libraries that use the port name already. We need to check it before merging this PR. should not take to long

**Comment by gaffneyd4** [comment]
@clamoriniere can you advise on what next steps are required for this PR?

**Comment by gaffneyd4** [comment]
@clamoriniere gentle bump on this PR. We have been kustomize-patching this change into our production environment since I opened the PR last July and don't seem to be encountering any issues. Is there anything I can do to carry this change forward?

**Review by vboulineau — APPROVED** [review]

**Comment by vboulineau** [comment]
Merged manually in https://github.com/DataDog/helm-charts/commit/81f4f64192a79c43ad5de4ff4bd46aee46dd2e58
To save you another rebase.

**Comment by gaffneyd4** [comment]
Thanks!

**Pull request closed without merging**
